src/libraries/grommet/components.js

```diff
@@ -2050,37 +2091,57 @@ export const components = {
   DataFilters: {
     component: DataFilters,
     name: 'DataFilters',
+    documentation: 'https://v2.grommet.io/datafilters',
     defaultProps: {
       drop: true,
     },
     properties: {
       drop: true,
+      heading: '',
       layer: false,
+      clearFilters: true,
+      updateOn: ['change', 'submit'],
     },
   },
   DataFilter: {
     component: DataFilter,
     name: 'DataFilter',
-    properties: {},
+    documentation: 'https://v2.grommet.io/datafilter',
+    properties: {
+      property: '',
+    },
   },
   DataSearch: {
     component: DataSearch,
     name: 'DataSearch',
-    properties: {},
+    documentation: 'https://v2.grommet.io/datasearch',
+    defaultProps: {
+      drop: true,
+      responsive: true,
+    },
+    properties: {
+      drop: true,
+      responsive: true,
+    },
   },
   DataSort: {
     component: DataSort,
     name: 'DataSort',
-    properties: {},
+    documentation: 'https://v2.grommet.io/datasort',
+    properties: {
+      drop: true,
```

Comment:
```suggestion
      drop: false,
```
This is the default behavior.